### PR TITLE
Feature/memory store

### PR DESCRIPTION
**Pull request by jaquadro**

### CODE REVIEW

File: test/util/TestServices.js [hunk 1]
```diff
@@ -4,11 +4,14 @@ const bigInt = require('big-integer');
 const kinesalite = require('kinesalite');
 const dynalite = require('dynalite');
 const AWS = require('aws-sdk-q');
-const http = require('http');
+const https = require('https');
+
+// Kinesalite is self-signed and normally invalid for node, so ignore it.
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
```
Comment (gotrevor) [inline test/util/TestServices.js]: This looks better than turning off SSL.  But, why did you need to make this change?  Maybe a Windows vs. OS X difference?

Comment (jaquadro) [inline test/util/TestServices.js]: Because my node will not run any code with a self-signed certificate without setting that.
